week-1/mandatory/2-classes-db/task.md

```diff
@@ -2,13 +2,21 @@
 
 ## Submission
 
-Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week.  You will find it beneficial to complete the reading tasks before attempting some of these.
+Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week. You will find it beneficial to complete the reading tasks before attempting some of these.
 
 To submit this homework write the correct commands for each question here:
 
 ```sql
-
-
+1. select * from rooms where rate > 100;
+2. select * from reservations where (checkin_date >= '2020-09-01' and checkin_date <='2020-09-30') and checkout_date - checkin_date < 3;
+3. select * from customers where city like 'M%';
+4. insert into room_types values('PENTHOUSE','185.00');
+5. insert into rooms (room_no, rate, room_type) values(501,(select def_rate from room_types where room_type = 'PENTHOUSE'),'PENTHOUSE'),(502,(select def_rate from room_types where room_type = 'PENTHOUSE'),'PENTHOUSE');
+6. insert into rooms (room_no, rate, room_type) values(503,143.00,'PREMIER PLUS');
+7. select all from reservations where checkin_date >= '2020-08-01' and checkin_date <= '2020-08-31'; / select count(id) from reservations where checkin_date >= '2020-08-01' and checkin_date <= '2020-08-31';
```

Comment:
Not sure what you expect with `select all from ...` - it doesn't produce any useful results.
To catch reservations that overlap the ends of the month you could try using:
`... WHERE checkin_date <= '2020-08-31` and checkout_date > '2020-08-01'

Reply:
Oh, yes that's more like it. I never thought of using the checkout date as well in my where clause.
Thanks for that.